packages/css/src/components/Popover.css

```diff
@@ -0,0 +1,30 @@
+.Popover {
+  --spaceAround: var(--space-s);
+
+  opacity: 0;
+  transition: opacity 0.1s ease-out;
```

Comment:
Question: isn't `0.1s` too fast for an animation? I usually use .2s+.

Reply:
I'll see that with Juan (designer in charge) during the final QA phases